src/mapeo-project.js

```diff
@@ -727,10 +727,10 @@ export class MapeoProject extends TypedEmitter {
 
     try {
       // check for already present fields and presets and delete them if exist
-      await deleteAll(this.preset)
-      await deleteAll(this.field)
+      const deletePresetsPromise = deleteAll(this.preset)
+      const deleteFieldsPromise = deleteAll(this.field)
       // delete only translations that refer to deleted fields and presets
-      await deleteTranslations({
+      const deleteTranslationsPromise = deleteTranslations({
```

Comment:
I don't think this is right. This will kick off the deletions, and it will only _wait_ for them if config import completes...but it will still delete them.

I think you need to only delete things at the very end.